js/ui/codingManager.js

```diff
@@ -380,7 +460,24 @@ const CodingSession = new Lang.Class({
                                            '',
                                            GObject.ParamFlags.READWRITE |
                                            GObject.ParamFlags.CONSTRUCT_ONLY,
-                                           WindowTrackingButton.$gtype)
+                                           WindowTrackingButton.$gtype),
+        'install-monitor': GObject.ParamSpec.object('install-monitor',
+                                                    '',
+                                                    '',
+                                                    GObject.ParamFlags.READWRITE |
+                                                    GObject.ParamFlags.CONSTRUCT_ONLY,
+                                                    CodingInstallationMonitor.$gtype)
+    },
+    Signals: {
+        'ready': {},
+        'closed': {},
+        'flipped': {
+            param_types: [ GObject.TYPE_INT ]
+        }
+    },
+
+    get state() {
```

Comment:
Do you want this to be a property instead? (and remove the `flipped` signal, since you could notify on it)